### ISSUE TITLE
t/000-compile-modules.t Error: can't open (eval 6) for input

### ISSUE THREAD

**Issue by d-lamb** [ingydotnet/module-compile-pm#5]
On a Mac OS X VM (10.10.1), using the system Perl (5.18.2), installation of Module::Compile (via CPAN and manually) fails because t/000-compile-modules.t fails.

The failure is exactly the same as in, e.g., http://www.cpantesters.org/cpan/report/b3617498-ded7-11e4-aa8d-adc4dc153cf8

```
#   Failed test 'use Module::Optimize;'
#   at t/000-compile-modules.t line 9.
#     Tried to use 'Module::Optimize'.
#     Error:  Can't open (eval 6) for input:
# No such file or directory at <snip>/blib/lib/Module/Compile.pm line 201.
# BEGIN failed--compilation aborted at (eval 6) line 2.

#   Failed test 'use Module::Compile;'
#   at t/000-compile-modules.t line 9.
#     Tried to use 'Module::Compile'.
#     Error:  Can't open (eval 7) for input:
# No such file or directory at <snip>/blib/lib/Module/Compile.pm line 201.
# BEGIN failed--compilation aborted at (eval 7) line 2.
# Looks like you failed 2 tests of 3.
t/000-compile-modules.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/3 subtests 
```

**Comment by SR71A** [comment]
Same platform and same exact error. 
Please address.

**Comment by wbirkett** [comment]
I had the same problem on Mac OSX 10.11.5. It was due to a bug in v0.98 of Test::Simple, as used by Apple. I installed the latest version of Test::Simple and the module passed all tests.

**Comment by ingydotnet** [comment]
Fixed in v0.36
